### PR TITLE
Drip megapack 100% real not fake 1 commit church buff speedmerge

### PR DESCRIPTION
**Pull request by ThePainkiller**
## What's this about a church buff...?

- Adds/Improves Justice, Pandemonica, Malina, Zdrada and Modeus alt styles for charming outfit
- Adds a black suit jacket with adjustable styles for the charming outfit
- Adds charming waistcoat for Malina/Cerberus drip
- Ports the Tojo Pants and Tojo Jacket (Yakuza reference)
- Ports black and red overcoat styles (Persona 5, Saints Row)
- New sprites for Prospector/Foreman lockers, Salvager lockers changed in appearance to rusty old ones for fluff
- Adds New Testament Knight Hardsuit RIG module for the church's New Testament Armaments Disk, a Divisor/Numerical joint effort in producing a combat-ready hardsuit. A pre-loaded version with flash, shield, jetpack and storage modules spawns on the Prime's office's altar. Same stats as the Combat Hardsuit.
- EDIT: SCAF helmet and Blackshield SCAF helmet now have cat eared alt styles.

<hr>

## Sprites preview:

image

image

image

<hr>

## Changelog
:cl:
add: Added new styles for the charming outfit. Impersonate ALL of the demons!
add: Added a stylish black suit jacket, with alternate styles for more drip
add: Added Tojo clan Pants and Jacket, become a Mad Dog of Shimano today!
add: Ported Black and Crimson coats from Skyrat (Joker/Morningstar)
add: Added New Testament "Knight" Hardsuit module, a combat hardsuit for the church available on the New Testament Armaments disk. A pre-equipped version spawns on the Prime's office.
add: SCAF and Blackshield SCAF now with cat ear alt styles.
imageadd: New sprites for Prospector and Foreman lockers. Scavengers locker sprites changed to old lockers for fluff.
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/suits/combat.dm [hunk 1]
```diff
@@ -106,3 +106,57 @@
...
+	name = "knight hardsuit gauntlets"
+
+/obj/item/clothing/shoes/magboots/rig/combat/knight
+	name = "knight hardsuit greaves"
+
+/obj/item/clothing/head/helmet/space/rig/combat/knight
+	name = "knight hardsuit full helm"
+	desc = "One's spirit is generally the greatest shield."
+	light_overlay = "helmet_light_dual"
+	armor_list = list(
```
Comment (r4d6) [inline code/modules/clothing/spacesuits/rig/suits/combat.dm:155]: The RIG overwrite all armor, not just the ones that aren't defined.

Comment (ThePainkiller) [reply code/modules/clothing/spacesuits/rig/suits/combat.dm:155]: Does that mean the blackshield hood with its armor values defined above is moot as well?

Comment (r4d6) [reply code/modules/clothing/spacesuits/rig/suits/combat.dm:155]: Well, I'm not 100% sure if that's the case, but I do know that it was the case for the Hydrogen Knight Rig Suit's chest pieces.

**Comment by BigBadPossum** [comment]
Conflicts.

**Comment by ThePainkiller** [comment]
Good to go now.

**Comment by Trilbyspaceclone** [comment]
going to have to do versons for ***all*** scav armor if you kot them

**Comment by Trilbyspaceclone** [comment]
Ill fix it myself but possum already wanted to merge it so ill do it for sake of getting bug fixes out faster

**Pull request merged**